packages/ui/src/components/ui/attachment/attachmentAdmin/attachmentAdmin.tsx

```
@@ -51,21 +61,22 @@ const AttachmentAdmin = (props: AttachmentProps) => {
             animate={{ opacity: 1, scale: 1 }}
           >
             <Card className="bg-[#26262666] p-4 border-0 flex gap-4">
-              <ScrollArea>
+              <div className="flex flex-row overflow-auto w-full gap-4">
                 {files.map((file) => (
                   <motion.div
                     exit={{ opacity: 0, scale: 1 }}
                     initial={{ opacity: 0, scale: 0.7 }}
                     animate={{ opacity: 1, scale: 1 }}
                   >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)